build-system/tsconfig.json

```diff
@@ -27,15 +27,50 @@
     "./common/**/*.ts",
     "./server/**/*.js",
     "./server/**/*.ts",
-    "./tasks/**/*.js",
-    "./tasks/**/*.ts"
+    "./tasks/*.js",
+    "./tasks/*.ts",
```

Comment:
Can we simplify this to `./tasks/*` ? Looks like the only supported extensions are `.ts, .tsx, .d.ts, .js, .jsx` anyways.
https://www.typescriptlang.org/tsconfig#include

Reply:
That is an excellent point.